string[] item

float32[] data